webapp/src/main/java/com/box/l10n/mojito/service/branch/BranchStatisticService.java

```diff
@@ -97,6 +99,11 @@ public class BranchStatisticService {
   @Value("${l10n.branchStatistic.quartz.schedulerName:" + DEFAULT_SCHEDULER_NAME + "}")
   String schedulerName;
 
+  @Value("${l10n.branchStatistic.branch.notify.regex:}")
```

Comment:
Should the default be ".*" instead?
I think the string defaults to "" which means no branches would match by default which seems unintuitive

Reply:
If it's an empty string the pattern isn't compiled and is not used to filter branches, so all branches are processed in that case